src/components/toast/Toast.tsx

```diff
@@ -1,15 +1,23 @@
 import React from 'react';
 
 import { motion } from 'framer-motion';
+import { CheckCircleDuotoneIcon } from 'src/icons/CheckCircleDuotoneIcon';
+import { InfoDuotoneIcon } from 'src/icons/InfoDuotoneIcon';
+import { RemoveCircleDuotoneIcon } from 'src/icons/RemoveCircleDuotoneIcon';
+import { WarningDuotoneIcon } from 'src/icons/WarningDuotoneIcon';
+import { theme } from 'src/styles/constants/theme';
+import { Intent } from 'src/types';
 import styled from 'styled-components';
 
 const AminoToast = styled(motion.div)`
-  background: var(--amino-gray-700);
+  background: ${theme['gray-l80']};
   z-index: 999999;
   border-radius: var(--amino-radius-lg);
   color: white;
   box-shadow: var(--amino-shadow-medium);
   padding: var(--amino-space-half) var(--amino-space);
```

Comment:
I would like the rest of the team to weigh in on this decision: https://app.frontapp.com/open/cnv_ez8da8b

Reply:
Thanks for catching that Thien! I'd forgotten to update the other variables. Once we decide will camel or snake, I can make the changes in this file

Reply:
@Thienhuynh95 It looks like everyone is good with moving the variables to snake-case. Now all we need is a good regex or script to update all of dashboards variables 🤷‍♂️

Reply:
I just want to make sure that I got it right. Do you mean `test-abc` or `testAbc`?

Reply:
Hah! How embarrassing. 🙄   I made a mistake in my previous comment (I have since updated it). `testAbc` is what we want.

Reply:
Sounds good to me, I'll update the script to generate the camelCase variable and then update all variables in amino first and then dashboard.
@devaneereid I think we don't need to update these variables for now since that will be changed later anyway. I'll update them all after I finished the building script